azerothcore/azerothcore-wotlk
# Core/Scripts: Fixed Mimiron's Spinning Up + proper timers (wowwiki)

## CHANGES PROPOSED:
-  Blizzlike timers (as of wowwiki) for the spinning up event.
-  Proper check for the spinning up position so there's no delay.


## ISSUES ADDRESSED:
- Closes #1611



## TESTS PERFORMED:
- Take him in P2 - check if the position of the laser beam is accurate
- Take him in P4 - check if the position of the laser beam is accurate



## HOW TO TEST THE CHANGES:

- Take him in P2 - check if the position of the laser beam is accurate
- Take him in P4 - check if the position of the laser beam is accurate

## KNOWN ISSUES AND TODO LIST:

- None


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
